src/elements/content-sidebar/activity-feed/task-new/Assignees.js

```diff
@@ -53,33 +62,42 @@ class Assignees extends React.Component<Props> {
         const areThereMoreEntries = !!next_marker; // there are more assignees in another page of results
         const visibleAssignees = entries
             .slice(0, maxAvatars)
-            .map(({ id: assignmentId, target, status: assigneeStatus }) => {
+            .map(({ id, target, status, completed_at: completedAt }) => {
                 return (
-                    <Tooltip key={assignmentId} text={<AssigneeTooltipLabel user={target} status={assigneeStatus} />}>
+                    <Tooltip
+                        key={id}
+                        text={<AssigneeTooltipLabel user={target} status={status} completedAt={completedAt} />}
+                    >
                         <AssigneeStatus
-                            key={assignmentId}
```

Comment:
Is `key` no longer needed or was it not needed originally?

Reply:
There is a key in the outer wrapper, doesn't need to be on this element

Reply:
Understood: it wasn't needed originally. Thanks for validating.